airbyte_cdk/sources/declarative/yaml_declarative_source.py

```diff
@@ -19,10 +19,11 @@ class YamlDeclarativeSource(ConcurrentDeclarativeSource[List[AirbyteStateMessage
     def __init__(
         self,
         path_to_yaml: str,
-        debug: bool = False,
         catalog: Optional[ConfiguredAirbyteCatalog] = None,
         config: Optional[Mapping[str, Any]] = None,
         state: Optional[List[AirbyteStateMessage]] = None,
+        *,
+        debug: bool = False,
```

Comment:
@coderabbitai - I've reverted this change for now as it's outside the scope of the current PR. However, because it was surfaced as a linting/typing issue, we should circle back to it eventually. Could you create an issue for it?